spec/vast_trackers.spec.js

```diff
@@ -282,5 +285,41 @@ describe('VASTTracker', function() {
         expect(spyTrack).not.toHaveBeenCalledTimes(2);
       });
     });
+    describe('#error', () => {
+      let isCustomCode = false;
+      beforeEach(() => {
+        vastTracker.error();
+      });
+      it('should be defined', () => {
+        expect(ad.errorURLTemplates).toBeDefined();
+      });
+      it('should have emitted an ad Error and called trackUrl ', () => {
+        expect(spyTrackUrl).toHaveBeenCalledWith(
+          ['http://example.com/error_[ERRORCODE]'],
+          expect.objectContaining(expectedMacros),
+          expect.objectContaining({ isCustomCode })
+        );
+        expect(spyTrackUrl).toHaveBeenCalledTimes(1);
+      });
+    });
+    describe('#errorWithCode', () => {
+      beforeEach(() => {
+        vastTracker.errorWithCode();
+      });
+      it('should be defined', () => {
+        expect(ad.errorURLTemplates).toBeDefined();
```

Comment:
`errorURLTemplates` have nothing to do with `errorWithCode` function, this assert is not necessary.